it/ch00.md

```diff
@@ -0,0 +1,35 @@
+# Prefazione
+Questa guida è open source e manutenuta da ErCiccione, della Comunità Monero. Se hai suggerimenti o vuoi contribuire allo sviluppo di questa guida, puoi aprire una pull request o Issue sul repositorio GitHub dove viene sviluppato questo documento: [github.com/monero-ecosystem/monero-GUI-guide](https://github.com/monero-ecosystem/monero-GUI-guide).
```

Comment:
I think the first phrase should not contain a comma. "appartenente alla comunità Monero"..

Reply:
Agree with "appartenente", less with the absence of comma.

Reply:
Membro is perfect!